server/src/emails/album-update.email.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Body,
+  Button,
+  Column,
+  Container,
+  Head,
+  Hr,
+  Html,
+  Img,
+  Link,
+  Preview,
+  Row,
+  Section,
+  Text,
+} from '@react-email/components';
+import * as CSS from 'csstype';
+import * as React from 'react';
+import { AlbumUpdateEmailProps } from 'src/interfaces/notification.interface';
+
+export const AlbumUpdateEmail = ({ baseUrl, albumName, recipientName, albumId, cid }: AlbumUpdateEmailProps) => (
+  <Html>
+    <Head />
+    <Preview>You have added to a shared album.</Preview>
```

Comment:
Copy paste mistake?

Reply:
I think it should say New media has been added to <album>

Reply:
```suggestion
    <Preview>New media has been added to a shared album.</Preview>
```